### ISSUE TITLE
error: file not found for module `config`

### ISSUE THREAD

**Issue by tdz** [fabricedesre/cc3200-rs#53]
Git ref 5817f0d2b346458fc4029497eef0411fd9e4f256 produces the following error message:

   Compiling cc3200 v0.1.0 (file:///home/mozilla/Projects/mozilla/src/sensorweb/cc3200-rs)
error: file not found for module `config`
  --> src/lib.rs:42:9
   |
42 | pub mod config; // SSID and password currently set here
   |         ^^^^^^
   |
   = help: name the file either config.rs or config/mod.rs inside the directory "src"

error: aborting due to previous error
error: Could not compile `cc3200`.

Forgot to check-in a file? I suggest to make the comment a bit more verbose, if users are supposed to create config.rs and set SSID and password there.

**Comment by fabricedesre** [comment]
Yes, you need to copy config.rs.sample to config.rs and edit it to include your network credentials. We did the same in the sensorweb-firmware repository as a safeguard against people committing their passwords.

**Comment by fabricedesre** [comment]
Actually, this line needs to be removed from src/lib.rs
